.dockerignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
Yea. That one fix nearly halved the amount of data sent to the docker daemon during a docker build.